docs/topics/ref_new-features-7-1-0.adoc

```diff
@@ -0,0 +1,104 @@
+
+For more information about the support scope of Red{nbsp}Hat Developer Preview software, see link:https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
+====
+
+
+
+.Assessment and review statuses are now displayed in the MTA UI
+
+Previously, the status of assessment and review was only displayed for applications. With this update, the status of the archetype assessment and review processes is displayed in the {ProductFullName} user interface (UI):
+
+* The assessment statuses are the following:
+** Completed: All required assessments completed.
+** InProgress: The assessment process is in progress.
+** NotStarted: The assessment process has not been started.
+* The review statuses are the following:
+** Completed: A review exists.
+** NotStarted: The review process has not been started.
+
+
+
+.New feature Insights has been added
+
+Issues that do not impact the migration but contain useful information about the technologies used in the application and their usage in the code are categorized as Insights. Insights do not have an effort and category assigned, but might have a message and tag. You can view Insights in the Static report under the Insights tab and in the analysis output under the Insights section. 
+
+
+
+.A new  Task Manager tab is now available in the MTA user interface
+
+Previously, tasks that were being executed and the queue of pending tasks were not displayed in the {ProductFullName} user interface (UI). With this update , a new Task Manager tab is available to view the following information about the tasks that are in the queue for execution:
+
+* *ID*: The ID of the task.
+* *Application*: The application name associated with the task.
+* *Status*: The status of the task, for example, *Scheduled*, *Pending*, *In progress*, *Succeeded*, or *Failed*.
+* *Kind*: The type of the task, for example, *analyzer* or *discovery*.
+* *Priority*: Priority of the task. The value is from zero to any positive integer. The higher the value in this column, the higher the priority of the task.
+* *Preemption*: It allows the scheduler to cancel a running task and free the resources for higher priority tasks. The values are *true* or *false*.
+* *Created By*: The name of the user who created the task.
+
+
+
+.Multiple applications can now be selected to filter the applications list in Application Inventory
+
+Previously, you could select only one application to filter the results on the Application Inventory page. With this update, you can select multiple applications as a single filter to display the list of applications corresponding to this filter.
+
+
+
+.Support for providing a single report when analyzing multiple applications on the CLI
+
+{mta-cli} was designed to analyze a single application and produce a report about that application. With this update, you can use the `--bulk` option with the `analyze` command to analyze multiple applications and generate a single report for all of them.
```

Comment:
```suggestion
{mta-cli} was designed to analyze a single application and produce a report about that application. With this update, you can use the `--bulk` option of  the `analyze` command to analyze multiple applications, one `analyze` command per application,  but with a common output file for all of the reports. As described in the {UserCLIBookName}, this results in {mta-cli} generating a single analysis report for all the applications, instead of generating a separate report for each application.  
```